### PR TITLE
Setup eslint and cicle CI

### PR DESCRIPTION
**Pull request by sravan-s**
- Intro - Eslint v6.0
- Intro - Circle CI
- Push changes for Eslint
- Split eslint config to .eslintrc
- Update nodejs to LTS(10.16)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "description": "Apache Zeppelin web written in ReactJS",
   "engines": {
-    "node": ">=8.9",
+    "node": ">=10.16",
```
Comment (sravan-s) [inline package.json:6]: Updated nodejs to LTS

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
```
Comment (sravan-s) [inline .eslintrc:1]: Move eslintrc outside

File: src/components/Notebook.jsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React from 'react';
```
Comment (sravan-s) [inline src/components/Notebook.jsx:1]: I think airbnb rule gave me single quotes, is this cool?

Comment (AhyoungRyu) [reply src/components/Notebook.jsx:1]: I'm cool

**Review by AhyoungRyu — APPROVED** [review]
👏🏼👏🏼👏🏼

**Pull request merged**